lib/dirac_improved_staggered.cpp

```diff
@@ -122,6 +123,23 @@ namespace quda {
     // do nothing
   }
 
+  /* Creates the coarse grid dirac operator
```

Comment:
Make this doxgygen

Reply:
and move the comment to the header declaration

Reply:
These comments also reside in `dirac_wilson.cpp`, etc, effectively verbatim (i.e., without doxygen). If you'd like, I can remove them from both places.

In all cases there were doxygen comments in the header declaration already.

Reply:
In that case, make sure that any details from here and dirac_wilson.cpp are percolated to their respective comments in the header, and delete the comments here.  No need to duplication of comments, and the convention is to have details like this in the header.

Reply:
Will fix soon

Reply:
Done